client/my-sites/checkout/composite-checkout/record-analytics.js

```diff
@@ -251,6 +252,24 @@ export default function createAnalyticsEventHandler( reduxDispatch ) {
 				);
 			}
 
+			case 'EBANX_TRANSACTION_BEGIN': {
+				reduxDispatch(
+					recordTracksEvent( 'calypso_checkout_form_submit', {
+						credits: null,
```

Comment:
🤔  I wonder why we use `credits: null` in all these events? It seems like we might have credits. Was that a TODO item that we never got around to taking care of?

Reply:
This is not a blocker, FYI. We may just want to track it elsewhere.